File: app/src/main/java/com/material/androidpro/dashboard/DashboardModel.java
package com.material.androidpro.dashboard;

import com.material.androidpro.MaterialRetention;
import com.material.androidpro.compose.ui.ComposeUIRetention;

public class DashboardModel {

    private int image;
    private String title = "";
    private String shortDes = "";
    private @MaterialRetention.Material int materialType;

    private @ComposeUIRetention.ComposeUI int composeUIType;

    public int getImage() {
        return image;
    }

    public void setImage(int image) {
        this.image = image;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getShortDes() {
        return shortDes;
    }

    public void setShortDes(String shortDes) {
        this.shortDes = shortDes;
    }

    public int getMaterialType() {
        return materialType;
    }

    public void setMaterialType(@MaterialRetention.Material int materialType) {
        this.materialType = materialType;
    }

    public int getComposeUIType() {
        return composeUIType;
    }

    public void setComposeUIType(@ComposeUIRetention.ComposeUI int composeUIType) {
        this.composeUIType = composeUIType;
    }
}

File: app/src/main/java/com/material/androidpro/dashboard/DashboardModel.kt
package com.material.androidpro.dashboard

import com.material.androidpro.MaterialRetention.Material
import com.material.androidpro.compose.ui.ComposeUIRetention.ComposeUI

data class DashboardModel(

    @JvmField
    var image: Int = 0,

    @JvmField
    var title: String = "",

    @JvmField
    var shortDes: String = "",

    @Material
    var materialType: Int = 0,

    @ComposeUI
    var composeUIType: Int = 0

)
